Source/Models/JSONDeserializer.swift

```diff
@@ -1,15 +1,17 @@
 import Argo
 import Result
 
-struct JSONDeserializer: Deserializer {
-  func deserialize(_ data: Data?) -> Result<Any, SwishError> {
+public struct JSONDeserializer: Deserializer {
```

Comment:
Huh. Was this never public before?

Reply:
It never needed to be! I was actually a little surprised too.